test/helper.rb

```diff
@@ -25,22 +25,46 @@
 
 require_relative '../lib/irb-power_assert'
 
-class Test::Unit::TestCase
-  # Taken from https://github.com/ruby/irb/blob/ad08152c43d4309ee4dec3bbaf361ffc338c1f46/test/lib/minitest/unit.rb#L461-L495, thank you!
-  def capture_io
```

Comment:
I didn't know test-unit added in 14 years ago... https://github.com/test-unit/test-unit/commit/fd94c6b4bf201045293ed93fd3eda768aa514f1a